Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -112,6 +113,11 @@ private void OnInteractUsingEvent(EntityUid uid, FlammableComponent flammable, I
             if (!isHotEvent.IsHot)
                 return;
 
+            if (flammable.Ignitable)
+            {
+                flammable.FireStacks = Math.Max(2, flammable.FireStacks);
+            }
```

Comment:
this should be getting handled in `Ignite`, not only interact using. IMO `Ignitable` should be renamed to be something like `AlwaysCombustible`, since I think that fits better what this is going for

Reply:
actually, an even better solution would be to just make `FireStacks` serializable, and add an option to prevent `FireStacks` from ever decreasing (then just give paper permanent firestacks), which 1) has this same behavior in a more intuitive way and 2) allows more granularity with how 'always capable of being on fire' something is